lib/moonshot/command_line.rb

```diff
@@ -4,12 +4,23 @@ module Moonshot
   # This class implements the command-line `moonshot` tool.
   class CommandLine
     def self.run! # rubocop:disable AbcSize, CyclomaticComplexity, MethodLength, PerceivedComplexity
+      # Handling premature early command like
+      # `moonshot new` or `moonshot --help`
+      catch(:regular_command) do
+        handle_early_commands(ARGV[0..1])
+        return
+      end
+
       # Find the Moonfile in this project.
       orig_dir = Dir.pwd
 
       loop do
         break if File.exist?('Moonfile.rb')
-        raise 'Could not find Moonfile.rb!' if Dir.pwd == '/'
+
+        if Dir.pwd == '/'
+          raise 'No Moonfile.rb found, are you in a project? Maybe you need to '\
```

Comment:
I would warn the longer text out, and raise a simpler error. The output we do in the binstub takes the first line of the backtrace so I just think this would be hard to read.